### PR TITLE
follow and bootstrap tweaks

### PR DESCRIPTION
**Pull request by xithan**

### CODE REVIEW

File: card/lib/card.rb [hunk 1]
```diff
@@ -32,7 +32,8 @@ class Card < ActiveRecord::Base
 
   attr_accessor :action, :supercard, :current_act, :current_action, 
     :comment, :comment_author,    # obviated soon
-    :update_referencers,           # wrong mechanism for this
+    :update_referencers,          # wrong mechanism for this
+    :update_all_users,                  # if the above is wrong then this one too
```
Comment (ethn) [inline card/lib/card.rb]: heheh

File: card/lib/card/format.rb [hunk 2]
```diff
@@ -503,9 +503,13 @@ module Location
...
-        format = opts[:format] ? ".#{opts.delete(:format)}"  : ''
-        query  = opts.present? ? "?#{opts.to_param}"         : ''
-        card_path "#{title.to_name.url_key}#{format}#{query}"
+        card_path page_path_segment(title, opts)
+      end
+      
+      def page_update_path title, opts={}
+        Rails.logger.warn "Pass only Card::Name to page_update_path #{title.class}, #{title}" unless Card::Name===title
+        update_opts = opts[:card] ? opts : { :card => opts }
+        card_path "update/#{page_path_segment(title, update_opts)}"
```
Comment (ethn) [inline card/lib/card/format.rb]: Is this really necessary (especially for one call)?.   Can you do :path_opts=>{:action=>:update} and use page_path?

File: card/mod/05_email/set/type_plus_right/user/follow.rb [hunk 3]
```diff
@@ -152,6 +158,30 @@ def followed_by_set
...
+       end
+     end
+     #
+     # binding.pry
+     # save_interrupted_action(request.env['REQUEST_URI'])
+     # "You have to #{ link_to 'sign in', card_url(':signin') } #{to_task}"
+   end
+   
+   view :denial do |args|
+     binding.pry
```
Comment (ethn) [inline card/mod/05_email/set/type_plus_right/user/follow.rb]: binding.

File: card/lib/card/format.rb [hunk 4]
```diff
@@ -504,10 +504,11 @@ module Location
       def page_path title, opts={}
         Rails.logger.warn "Pass only Card::Name to page_path #{title.class}, #{title}" unless Card::Name===title
         format = opts[:format] ? ".#{opts.delete(:format)}"  : ''
+        action = ( (path_opts = opts.delete(:path_opts)) && path_opts[:action] ) ? "#{opts[:path_opts][:action]}/" : ''
         query  = opts.present? ? "?#{opts.to_param}"         : ''
-        card_path "#{title.to_name.url_key}#{format}#{query}"
+        card_path "#{action}#{title.to_name.url_key}#{format}#{query}"
```
Comment (ethn) [inline card/lib/card/format.rb]: ok, now I think I was getting confused between card_link (which uses "path") and card_path.  this is all pretty tangled still, I guess.

Comment (ethn) [inline card/lib/card/format.rb]: let's not resolve that in this branch ;)  sorry for my confusion

Comment (ethn) [inline card/lib/card/format.rb]: (meaning I'll merge as is unless I see other issues)

File: card/mod/05_email/set/type_plus_right/user/follow.rb [hunk 5]
```diff
@@ -134,13 +136,16 @@ def followed_by_set
...
+     end
+     never = Card[:never].name
      wrap_with :div, :class=>'pointer-list-editor' do
-       wrap_with :ul, :class=>'delete-list' do
+       wrap_with :ul, :class=>'delete-list list-group' do
          
          ignore_list.map do |rule_card|
            content_tag :li do
-             subformat(rule_card).render_item_delete :condition=>'ignore', :editable=>editable
+             subformat(rule_card).render_follow_item :condition=>never, :hide=>hide_buttons
```
Comment (ethn) [inline card/mod/05_email/set/type_plus_right/user/follow.rb]: safe to assume you've tested the ignore behavior in the browser?

Comment (xithan) [inline card/mod/05_email/set/type_plus_right/user/follow.rb]: yes

**Pull request merged**